salt/modules/hadoop.py

```diff
@@ -88,6 +88,28 @@ def dfs(command=None, *args):
         return 'Error: command must be provided'
 
 
+def dfsadmin_report(arg=None):
+    '''
+    Reports basic filesystem information and statistics. Optional flags may be used to filter the list of displayed DataNodes.
+
```

Comment:
This new function could use a `.. versionadded:: Fluorine` tag, if you wouldn't mind. :)

Reply:
@rallytime I use a .. versionadded:: Fluorine tag !